packages/bpk-component-input/src/withOpenEvents.tsx

```diff
@@ -67,15 +67,14 @@ type EventHandlers = {
 type InputProps = ComponentProps<'input'> &
   Omit<EventHandlers, 'readOnly' | 'aria-readonly'>;
 
-const handleKeyEvent =
-  (keyCode: number, callback?: () => void) => (e: UIEvent) => {
-    if (e instanceof KeyboardEvent && e.keyCode === keyCode) {
```

Comment:
e was never coming back as an instanceof Keyboard event even though it is a subtype of UIEvent so mean it was always evaluating false and never running this code further!